sktime/utils/utilities.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+from pandas import DataFrame
+
+# little accuracy helper function
+def accuracy(actual_class_values, predicted_class_values):
```

Comment:
There is an `accuracy_score` function in sklearn which you can use

Reply:
ah this was before I discovered that, I'll get rid of it as it's redundant now